modules/android/examples/codesnippet_collection.java

```diff
@@ -2687,16 +2702,15 @@ public class ReplicationExamples {
 
     public void activeReplicatorExample(Set<Collection> collections)
         throws URISyntaxException {
-        // tag::p2p-act-rep-start-full[]
         // Create replicator
         // Consider holding a reference somewhere
-        // to prevent the Replicator from being GCed
+        // to prevent the Replicator from being 
+        // tag::p2p-act-rep-func[]
         Replicator repl = new Replicator( // <.>
 
-            // tag::p2p-act-rep-func[]
+            
             // initialize the replicator configuration
-            new ReplicatorConfiguration(new URLEndpoint(new URI("wss://listener.com:8954"))) // <.>
-                .addCollections(collections, null)
+            new ReplicatorConfiguration(CollectionConfiguration.fromCollections(collections), new URLEndpoint(new URI("wss://listener.com:8954"))) // <.>
```

Comment:
This line is going to be too long on the website. Can the collections part be extracted to its own line?